src/main/java/temporal/SimpleWorkflow.java

```diff
@@ -0,0 +1,26 @@
+package temporal;
+
+import io.temporal.workflow.*;
+import temporal.model.OrderInfo;
+
+@WorkflowInterface
+public interface SimpleWorkflow {
+
+  @WorkflowMethod
+  void start();
+
+  @SignalMethod
+  void add(OrderInfo orderInfo);
+
+  @SignalMethod
+  void update(OrderInfo orderInfo);
+
+  @SignalMethod
+  void register(OrderInfo orderInfo);
```

Comment:
The register method is never used.  If it will not be used, then remove

Reply:
addressed